tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -33,9 +33,10 @@
 
 
 @pytest.mark.parametrize(
-    "deployment_type, per_unit_storage_throughput", [("PERSISTENT_1", 200), ("SCRATCH_1", None), ("SCRATCH_2", None)]
+    "deployment_type, per_unit_storage_throughput, auto_import_policy",
+    [("PERSISTENT_1", 200, "NEW_CHANGED"), ("SCRATCH_1", None, "NEW"), ("SCRATCH_2", None, "NONE")],
 )
-@pytest.mark.regions(["eu-west-1"])
+@pytest.mark.regions(["us-east-1"])
```

Comment:
Minor: Do we want to change the region here?

Reply:
no, we definitely don't. that was from me testing locally. good catch.

Reply:
done.